src/envo/envo-edit.owl

```diff
@@ -41163,6 +41165,24 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105204> (photosynthetic photon flux density)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Photosynthetically_active_radiation>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105204> "Number of photons in the 400-700 nm range received by a surface for a specified amount of time"@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105204> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105204> "2021-11-05T00:08:35Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_06105204> "PPFD"@en)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105204> "photosynthetic photon flux density"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105204> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001291> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> <http://purl.obolibrary.org/obo/ENVO_01001864>)))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105213> (photosynthetic photon flux)
```

Comment:
I'd expect this to be axiomatised with PATO:flux or [luminous flux](http://purl.obolibrary.org/obo/PATO_0001296)

The definition could use some improvement along the genus-differentia model.